tests/models/test_dag.py

```diff
@@ -739,13 +739,13 @@ def test_sync_to_db(self, mock_now):
         )
         with dag:
             DummyOperator(task_id='task', owner='owner1')
+            subdag = DAG('dag.subtask', start_date=DEFAULT_DATE, )
+            subdag.parent_dag = dag
+            subdag.is_subdag = True
```

Comment:
Is this change needed? Doesn't the SubDagOperator set these automatically? If not it should -- I don't think users should have to set these properties to get "working" subdags.

Reply:
I was set by DagBag. We don't use DagBag, so this value is not set.
We don't use DagBag, so this value is not set.

Reply:
Gotcha. Maybe worth a comment here just to explain why we need to set it in this case.

Reply:
I added a comment.